03-transactions.md

```diff
@@ -107,13 +107,13 @@ This output sends funds to the other peer, thus is a simple P2WPKH to `remotekey
 
 #### Offered HTLC Outputs
 
-This output sends funds to a HTLC-timeout transaction after the HTLC timeout, or to the remote peer on successful payment preimage.  The output is a P2WSH, with a witness script:
+This output sends funds to a HTLC-timeout transaction after the HTLC timeout, or to the remote peer using the payment preimage or the revocation key.  The output is a P2WSH, with a witness script:
 
     <remotekey> OP_SWAP
         OP_SIZE 32 OP_EQUAL
     OP_NOTIF
-        # To me via HTLC-timeout transaction (timelocked).
-        OP_DROP 2 OP_SWAP <localkey> 2 OP_CHECKMULTISIG
+        # To me via HTLC-timeout transaction (timelocked) or to you with revocation key.
+        OP_DROP 2 OP_SWAP <localkey> <revocationkey> 3 OP_CHECKMULTISIG
```

Comment:
Just noticed: doesn't this result in a race? Both `<local-sig> <revocation-sig>` and `<remote-sig> <revocation-sig>` can spend this output. At which point both endpoints could try to spend the output.